src/components/Button/style.ts

```diff
@@ -25,16 +27,31 @@ const StyledButton: React.FunctionComponent<Props> = styled.button`
    border-radius: 3px;
    outline: none;
    transition: background 0.2s, color 0.2s;
-   span {
-      position: relative;
-      left: 5px;
-      top: 1px;
-   }
+
    &:hover {
       background: ${(props: Props) =>
          props.color ? colors[props.color] : colors.primary};
       color: white;
       cursor: pointer;
+      .icon {
+         background-color: white;
+      }
    }
 `;
-export default StyledButton;
+
+export const SIcon = styled.span`
+   background-size: cover;
+   display: inline-block;
+   position: relative;
+   left: 5px;
+   top: 2.5px;
+   width: 1rem;
+   height: 1rem;
+   background-color: ${(props: Props) =>
+      props.color
+         ? onlineTheme.colors[props.color]

```

Comment:
you want to export the theme like that:
styled.span(({ theme }) => {
  return `
  background-color: ${(props: Props) =>
      props.color
         ? theme.colors[props.color]
  `;
});